server/index.js

```diff
@@ -41,6 +41,37 @@ app.post('/gallery/update', (req, res) => {
   }
 });
 
```

Comment:
Well Done here. I enjoy how the your CRUD operation utility methods clean up the code significantly.